PWGCF/Flow/Tasks/FlowTask.cxx

```diff
@@ -613,6 +639,9 @@ struct FlowTask {
       // magnet field dependence cut
       Magnetfield = getMagneticField(bc.timestamp());
     }
+    float independent = cent;
+    if (cfgUseNch)
+      independent = static_cast<float>(tracks.size());
```

Comment:
Do you expect only 90 Nch as maximum? 
That are the limits of your histograms

Reply:
No. 
The user should change the `axisIndependent` if he/she turn the` cfgUseNch` on
` ConfigurableAxis axisIndependent{"axisIndependent", {VARIABLE_WIDTH, 0, 5, 10, 20, 30, 40, 50, 60, 70, 80, 90}, "centrality axis for histograms"};`